roles/calibre-web/tasks/install.yml

```
@@ -176,6 +176,22 @@
     backup: yes
   when: not appdb.stat.exists
 
+name: Does /library/calibre-web/xklb-metadata.db exist?
```

Line 179 contains a bug:

@deldesir please don't announce that a PR was tested if (as in this case) that's not true.

```suggestion
- name: Does /library/calibre-web/xklb-metadata.db exist?
```

---

Apologies, I did not catch this as I made the upload and edits directly using Android's Chrome.